src/AdminConsole/Pages/Account/Login.cshtml.cs

```diff
@@ -23,7 +29,8 @@ public IActionResult OnGet(string? returnUrl = null)
             return LocalRedirect(returnUrl);
         }
 
-        EmailSent = TempData["EmailSent"] is not null;
```

Comment:
We replace this with an enumerable, so we can store the actual status and then display a message based of the current status in that temporary dictionary.